src/main/java/software/sigma/bu003/internship/coparts/controller/GlobalControllerExceptionHandler.java

```diff
@@ -0,0 +1,39 @@
+package software.sigma.bu003.internship.coparts.controller;
+
+import com.mongodb.MongoWriteException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import software.sigma.bu003.internship.coparts.service.exception.PartNotFoundException;
+
+import javax.validation.ConstraintViolationException;
+
+@ControllerAdvice
+public class GlobalControllerExceptionHandler {
+
+    @ExceptionHandler(PartNotFoundException.class)
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    public ResponseEntity<String> partNotFoundException(PartNotFoundException e) {
+        return ResponseEntity
+                .status(HttpStatus.NOT_FOUND)
+                .body(e.getMessage());
+    }
+
+    @ExceptionHandler(MongoWriteException.class)
+    @ResponseStatus(HttpStatus.ACCEPTED)
```

Comment:
202 Accepted - response status code indicates that the request has been accepted for processing, but the processing has not been completed.

I understand that the request is accepted, it is correct, but it will not be processed

Reply:
What status is usually used in such a case?

Reply:
500, but yes better don't hanlde MongoWriteException.class